### ISSUE TITLE
user-generated character

### ISSUE THREAD

**Issue by Vandivier** [Vandivier/arias-tale-rpg#28]
after #18 we will have a db schema for characters

we can use non-LLM approach to randomly roll a character using this approach, basically a randomized data approach

we can do template-based images. notably, users should not be able to roll a unique or ultra-rare character this way

in the future after #10 we can do GPT-4 bc users can pay and also we can let users buy these souls and have me manually roll and store them as well 

so, these will be basic characters and there is only one per account

in a new issue subsequently, people can buy bronze, silver, gold, and platinum souls manually ie by paying on Ladderly

in a further subsequent issue, people can buy here (with true GPT integration)

in a further subsequent issue, people can buy and sell on a marketplace

in a semi-related issue, owning a character will also generate a card

in another subsequent issue, characters can roll non-character encyclopedia things like places, etc

in a further issue, we will have decks of cards that can be bought together with the art, and maybe if you buy a deck you get a book for free? meh that sounds like a special deal we can do on occassion not the norm but we will see, depends on audience preference/willing to pay

**Comment by Vandivier** [comment]
this would be great to plugin to the GPT marketplace and could also be bound to crypto.
